examples/docs_snippets/docs_snippets/integrations/dbt/dbt_cloud.py

```diff
@@ -36,23 +36,36 @@ def scope_load_assets_from_dbt_cloud_job():
 
 
 def scope_schedule_dbt_cloud_assets():
-    dbt_cloud_assets = []
+    from dagster_dbt import dbt_cloud_resource, load_assets_from_dbt_cloud_job
+
+    dbt_cloud_instance = dbt_cloud_resource.configured(
+        {"auth_token": "foo", "account_id": 111}
+    )
+    dbt_cloud_assets = load_assets_from_dbt_cloud_job(
+        dbt_cloud=dbt_cloud_instance,
+        job_id=33333,
+    )
+
```

Comment:
Hmmm this is the api that we have, but we definitely need to rationalize the way that we handle our dbt resources